docs/modules/operation/pages/service-assurance/monitors/NrpeMonitor.adoc

```diff
@@ -30,15 +30,34 @@ CAUTION: {page-component-title} runs every NRPE check in a Java thread without `
 [options="header"]
 [cols="1,3,2"]
 |===
-| Parameter |Description                                                     | Default
+| Parameter
+| Description
+| Default
+
 3+|*Required*
-| command | The {\check_name} of the command configured as `command[{\check_name}]="/path/to/plugin/check-script`      | empty
+
+| command
+| The {\check_name} of the command configured as `command[{\check_name}]="/path/to/plugin/check-script`
```

Comment:
is this `{\check_name}` or `\{check_name}`?

Reply:
I'm not sure. I didn't change the text from what was originally there.

Reply:
Yes, you found a bug and your suggestion is correct. The `{check_name}` is a placeholder name for the NRPE check script running from the NRPE agent on the remote system, so `\{check_name}` is the right thing here.

Reply:
Thanks for verifying this!